benchmarks/basic_benchmarks.py

```diff
@@ -10,3 +11,134 @@ def setup(self):
 
     def time_solve_model(self):
         BasicBenchmark.sim.solve([0, 1800])
+
+
+class SmallPack:
+    def setup(self):
+        self.netlist = lp.setup_circuit(Np=2, Ns=1, Rb=1e-4, Rc=1e-2)
+        chemistry = pybamm.parameter_sets.Chen2020
+        self.parameter_values = pybamm.ParameterValues(chemistry=chemistry)
+
+    def time_discharge_1cpu(self):
+        experiment = pybamm.Experiment(
```

Comment:
Do you want to time the setting up of the experiment? If not, you can move it to setup so you are only timing the solving time.

Reply:
I don't think it adds much overhead compared to solve time but the idea was to vary the experiments for a given netlist so I put it in the methods. I guess I could define the different experiments in the setup and then pick one in the test